### PR TITLE
Flesh out overview

### PR DESCRIPTION
**Pull request by JesseTG**
Thoughts on some of  this so far?  It's not complete or proofread, but I think it's a start.

Also, this implements #6

### CODE REVIEW

**Comment by Bsabz** [comment]
I like this so far

File: design/requirements.md [hunk 1]
```diff
@@ -18,7 +18,29 @@ Discuss things we want to do, and things we *don't* want to do
 
 ### Definitions, acronyms, and abbreviations
 
-Anything not obvious
+**Work:** Any content a user has posted to Mixxy, typically an illustration.
+
+**Remix:** The act of one user making a copy of another's work for the purpose of editing, improving upon, or otherwise modifying the original work.  (The original work is *not* modified in the process; remixing cannot be used for vandalism.)
+
+**Content:** The subject matter of a work.
```
Comment (JesseTG) [inline design/requirements.md]: Ah, shit, this doesn't fit with the "Work" definition

Comment (JesseTG) [inline design/requirements.md]: You know what, after the first draft is written, I'll go through and proofread it; things like this will be fixed

**Comment by JesseTG** [comment]
I think this is ready to be merged, anyone else have thoughts?  If not, I'll merge it tomorrow night

**Comment by Bsabz** [comment]
Looks good to me

**Comment by cristianomj** [comment]
Looks good

**Comment by JesseTG** [comment]
Okey dokey.

**Pull request merged**